src/main/java/de/hysky/skyblocker/skyblock/slayers/SlayerEntitiesGlow.java

```diff
@@ -0,0 +1,117 @@
+                    new Box(entity.getX() - 0.4, entity.getY() - 0.1, entity.getZ() - 0.4, entity.getX() + 0.4, entity.getY() - 2.2, entity.getZ() + 0.4);
+            case SlayerUtils.TARA ->
+                    new Box(entity.getX() - 0.9, entity.getY() - 0.2, entity.getZ() - 0.9, entity.getX() + 0.9, entity.getY() - 1.2, entity.getZ() + 0.9);
+            case SlayerUtils.VOIDGLOOM ->
+                    new Box(entity.getX() - 0.4, entity.getY() - 0.2, entity.getZ() - 0.4, entity.getX() + 0.4, entity.getY() - 3, entity.getZ() + 0.4);
+            case SlayerUtils.SVEN ->
+                    new Box(entity.getX() - 0.5, entity.getY() - 0.1, entity.getZ() - 0.5, entity.getX() + 0.5, entity.getY() - 1, entity.getZ() + 0.5);
+            default ->
+                    new Box(entity.getX() - 0.5, entity.getY() + 0.1, entity.getZ() - 0.5, entity.getX() + 0.5, entity.getY() - 2.2, entity.getZ() + 0.5);
+        };
+    }
+
+    /**
+     * <p> Finds the closest matching MobEntity for the armorStand using entityClass and armorStand age difference to filter
+     * out impossible candidates, returning the closest mob of those remaining in the search box by block distance </p>
+     *
+     * @param entityClass the mob type of the Slayer (i.e. ZombieEntity.class)
+     * @param armorStand  the entity that contains the display name of the Slayer (mini)boss
+     */
+    private static MobEntity findClosestMobEntity(Class<? extends MobEntity> entityClass, ArmorStandEntity armorStand) {
+        return armorStand.getWorld().getEntitiesByClass(entityClass, armorStand.getDimensions(null).getBoxAt(armorStand.getPos()).expand(1.5), entity -> !entity.isDead() && entity.age > armorStand.age - 4 && entity.age < armorStand.age + 4)
+                .stream()
+                .filter(entity -> !(entity instanceof CaveSpiderEntity)) // CaveSpider extends Spider so filter out mob for BroodFather highlight
+                .min(Comparator.comparingDouble((MobEntity e) -> e.distanceTo(armorStand)))
+                .orElse(null);
+    }
+
+    /**
+     * <p> Adds the Entity UUID to the Hashset of Slayer Mobs to glow </p>
+     *
+     * @param armorStand the entity that contains the display name of the Slayer (mini)boss
+     */
+    public static void setSlayerMobGlow(ArmorStandEntity armorStand) {
+        String slayerType = SlayerUtils.getSlayerType();
+        Class<? extends MobEntity> entityClass = SLAYER_MOB_TYPE.get(slayerType);
+        if (entityClass != null) {
+            MobEntity closestEntity = findClosestMobEntity(entityClass, armorStand);
+            if (closestEntity != null) MOBS_TO_GLOW.add(closestEntity.getUuid());
+        }
+    }
+
+    public static void onEntityDeath(@Nullable Entity entity) {
+        if (entity != null && entity.getUuid() != null) MOBS_TO_GLOW.remove(entity.getUuid());
+    }
+
+    private static void clearGlow(Location location) {
+        MOBS_TO_GLOW.clear();
+    }
+
+    public static void init() {
```

Comment:
this method should go at the top to be consistent with the rest of the mod, also it should instead use the `JOIN` event from ClientPlayConnectionEvents or whatever its called